### ISSUE TITLE
extension of Pathlib.path break other libraries

### ISSUE THREAD

**Issue by daviddemeij** [fastai/fastcore#95]
The `pathlib.Path` extension [here](https://fastcore.fast.ai/utils.html#Extensions-to-Pathlib.Path) `read` and `write` break opening or writing using `pathlib.Path` paths in other libraries (for example `numpy` or `rasterio`). I also found someone else experiencing this issue on the fastai forums:
https://forums.fast.ai/t/solved-issue-with-utf-8-codec-when-saving-and-or-loading-np-ndarray/69072/2

This is quite an annoying issue since even importing a simple function from `nbdev` already patches `pathlib.Path`. Right now my solution is to delete these functions if they exist, but it is not very elegant:

```
if hasattr(Path, 'read'):
    del Path.read
if hasattr(Path, 'write'):
    del Path.write
```

I am not sure where the `Path.read()` and `Path.write()` extensions are used so it is hard for me to propose a good solution, but perhaps the extensions can be renamed to avoid conflicts with other libraries.

Example of this error occurring using `rasterio`:
![image](https://user-images.githubusercontent.com/19728972/94249612-6f537c00-ff20-11ea-8796-10feeaf947bc.png)

**Comment by daviddemeij** [comment]
Sorry, I found this is a duplicate issue: https://github.com/fastai/fastcore/issues/29 although it isn't resolved properly in my opinion. I agree that the issue is in other libraries checking if something is an open file by checking if it has a `read()`  or `write()` function, but it is not very user-friendly as it is now as it causes an error that can be very hard to debug if you are unaware of the `fastcore` patching done to `pathlib.Path`.

**Comment by jph00** [comment]
Many thanks for the report and the link to the previous issue. I hadn't seen before that someone had suggested a way to subclass `pathlib.Path` - something I've tried to do before but had many problems. I'll see if that works. Otherwise I'll see if we can rename these methods to something less common, since it does seem like plenty of libraries are making this silly assumption! :)

**Comment by jph00** [comment]
Fixed in v1.1
